api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -53,7 +53,7 @@
 	Condition getConditionByUuid(String uuid) throws APIException;
 
 	/**
-	 * Gets a patient's active conditions
+	 * Gets all conditions related to the specified patient.
```

Comment:
Did you intentionally drop the word active from the method description?

Reply:
@dkayiwa  Sorry, it was accidental. I updated the PR. Thank you :)